### PR TITLE
Basic server

### PR DESCRIPTION
**Pull request by ABSLord**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
+flake8==3.5.0
```
Comment (tmcgroul) [inline requirements-dev.txt]: Do we have different linter-setup on different projects?

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"""File for store fixtures. """
+
+import pytest
+from sanic.websocket import WebSocketProtocol
+
+from app.manager import make_app
+
+
+@pytest.yield_fixture
```
Comment (tmcgroul) [inline tests/conftest.py]: yield_fixture deprecated
https://docs.pytest.org/en/latest/yieldfixture.html

File: requirements-dev.txt (hunk 1 above)
Comment (ABSLord) [reply requirements-dev.txt]: done

File: tests/conftest.py (hunk 2 above)
Comment (ABSLord) [reply tests/conftest.py]: done

**Pull request merged**